app/views/admin/basic_projects/index.html.slim

```diff
@@ -0,0 +1,33 @@
+- content_for(:title, "Projects")
+
+= render "basic_projects_grid"
+
+- unless @empty_list
+  header.block.index-table
+    .filters
+      h4 = t("filter_by")
+
+      .larger-text-box.filter
+        label = t('activerecord.attributes.basic_project.name')
+        .filter-content
+          = grid_filter(@basic_projects_grid, :name_filter)
+
+      .filter
+        label = t('activerecord.attributes.basic_project.status')
+        .filter-content
+          = grid_filter(@basic_projects_grid, :status_filter)
+
+      .filter
+        label = t("activerecord.attributes.basic_project.start_date")
+        .filter-content.dates
+          = grid_filter(@basic_projects_grid, :start_date_filter)
+
+      .actions
+        button.btn.btn-link.wg-external-reset-button data-grid-name="basic_projects"
+          = t("clear")
+
+        button.btn.btn-primary.wg-external-submit-button.action data-grid-name="basic_projects"
+          = t("filter")
+      .clearfix
+
+= render_index_grid(@basic_projects_grid)
```

Comment:
I'm noticing here that there are two lines of code in this file (and all our grid index files) that seem to say "render index grid". Should we rename all the grid partials to e.g. `basic_projects_grid_spec` or `basic_projects_grid_definition`? Then it would be clear that the first render is for the definition only, while the second is where the code actually comes in...?